common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -770,6 +770,16 @@
   "screens.wynntils.characterSelection.playButton.description": "You can also double click to play",
   "screens.wynntils.characterSelection.playButton.play": "[>] Play with the selected character",
   "screens.wynntils.gearViewer.viewStats": "View Player's Statistics",
+  "screens.wynntils.guildMap.cycleDefenseFilter.description1": "Click here to cycle the territory defense filter",
+  "screens.wynntils.guildMap.cycleDefenseFilter.description2": "Hold SHIFT to filter higher, CTRL to filter lower",
+  "screens.wynntils.guildMap.cycleDefenseFilter.description3": "Current value: %s",
+  "screens.wynntils.guildMap.cycleDefenseFilter.high": "§cHigh",
+  "screens.wynntils.guildMap.cycleDefenseFilter.low": "§aLow",
+  "screens.wynntils.guildMap.cycleDefenseFilter.medium": "§eMedium",
+  "screens.wynntils.guildMap.cycleDefenseFilter.name": "Cycle Territory Defense Filter",
+  "screens.wynntils.guildMap.cycleDefenseFilter.off": "§7Off",
+  "screens.wynntils.guildMap.cycleDefenseFilter.veryHigh": "§cVery High",
+  "screens.wynntils.guildMap.cycleDefenseFilter.veryLow": "§aVery Low",
```

Comment:
These are unused (except `"screens.wynntils.guildMap.cycleDefenseFilter.name"`)? Are these meant to be used in the enum? If so, dont include the color codes in the localization

Reply:
Not meant to be used, was originally going to use them but the GuildResourceValues enum doesn't use them so the current comparison will break if someone switches languages